backend/ctgov/api/views.py

```diff
@@ -69,240 +82,219 @@ def get(self, request, *args, **kwargs):
 # This is for the front-end user interface control to display country drop-down
 class CountriesListApiView(APIView):
     def get(self, request, *args, **kwargs):
-        countries = (
+        qs1 = (
             Facilities.objects.all()
             .distinct()
+            .values("country")
+            .exclude(country="")
+            .exclude(country="United States")
             .order_by("country")
+        )
+        qs2 = (
+            Facilities.objects.all()
+            .distinct()
+            .filter(country="United States")
```

Comment:
Since there are performance issues on the AWS hosted site, I would highly recommend hard coding this value instead of an additional database pull for a known value.